docs/articles/basics/first_bot.md

```diff
@@ -62,7 +62,7 @@ Package|Description
 
 <br/>
 We'll only need the `DSharpPlus` package for the basic bot we'll be writing in this article.<br/>
-Select it from the list then click the `Install` button to the right.
+Select it from the list then click the `Install` button to the right (after verifing that you will be installing the 4.x RC version).
```

Comment:
Instead of 4.x RC you can probably just say "the latest 4.x version".

Reply:
I have updated the to say the latest 4.0 version in bold